molecule/default/tasks/exec.yml

```diff
@@ -0,0 +1,50 @@
+---
+- vars:
+    ns: k8s-exec
+    pod: sleep-pod
+    pod_def:
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        name: "{{ pod }}"
+        namespace: "{{ ns }}"
+      spec:
+        containers:
+          - name: sleeper
+            image: busybox
+            command: ["sleep", "infinity"]
+
+  block:
+    - name: Ensure that k8s-log namespace exists
```

Comment:
k8s-exec, or maybe just `{{ ns }}` so it's always accurate.

Reply:
Done